### PR TITLE
Add large bucket type and use new endpoint of c14 ls command

### PR DESCRIPTION
**Pull request by agirot**
Before with `c14 ls -a`

>  /storage/c14/safe
> > for : /storage/c14/safe/{safe_id}
>  > >for /storage/c14/safe/{safe_id}/archive/{archive_id}

Use new api endpoint :
>  /storage/c14/archive
> Returns a list of user's archives (with safe parent).

### CODE REVIEW

File: pkg/commands/ls.go [hunk 1]
```diff
@@ -108,51 +102,31 @@ func (l *ls) displayArchives(val []api.OnlineGetSafe) {
...
-		if err == nil {
-			sort.Sort(api.OnlineGetArchives(archives))
-			for j := range archives {
-				if l.flQuiet {
-					if l.flAll {
-						fmt.Fprintf(w, "%s %s\n", archives[j].UUIDRef, val[i].UUIDRef)
-					} else {
-						fmt.Fprintf(w, "%s\n", archives[j].UUIDRef)
-					}
+	if err == nil {
```
Comment (QuentinPerez) [inline pkg/commands/ls.go]: bad practice, golang community prefers 
```go
if err != nil {
    // do something
    return ...
}

File: pkg/commands/ls.go [hunk 2]
```diff
@@ -108,51 +102,31 @@ func (l *ls) displayArchives(val []api.OnlineGetSafe) {
...
+		sort.Sort(api.OnlineGetArchives(archives))
+		for j := range archives {
+			archive := archives[j]
```
Comment (QuentinPerez) [inline pkg/commands/ls.go]: use `for j, archive := archives`

File: pkg/commands/ls.go [hunk 3]
```diff
@@ -69,18 +68,15 @@ func (l *ls) Run(args []string) (err error) {
 		}
 		l.displayPlatforms(val)
 	} else {
-		l.OnlineAPI.CleanUpCache()
```
Comment (QuentinPerez) [inline pkg/commands/ls.go]: sure ?

**Comment by QuentinPerez** [comment]
can you update the readme with the newest flags ?

**Comment by agirot** [comment]
@QuentinPerez thx for feedback, i 'll rewrite your advice in my next pr !

File: pkg/api/methods.go [hunk 4]
```diff
@@ -92,6 +92,14 @@ func (o *OnlineAPI) GetArchives(uuidSafe string, useCache bool) (archives []Onli
 	return
 }
 
+func (o *OnlineAPI) GetAllArchives() (archives []OnlineGetArchive, err error) {
+	if err = o.getWrapper(fmt.Sprintf("%s/storage/c14/archive", APIUrl), &archives); err != nil {
+		err = errors.Annotate(err, "GetArchives")
+		return
+	}
```
Comment (QuentinPerez) [inline pkg/api/methods.go]: add the archives in the local cache, see https://github.com/agirot/c14-cli/blob/a55e90aa84086331b5aaf0ce9c61fba4f7fcdd7e/pkg/api/methods.go#L89-L92

**Pull request merged**